manifests/kots-config.yaml

```diff
@@ -724,6 +724,13 @@ spec:
           when: '{{repl (ConfigOptionEquals "aiApiEnabled" "1") }}'
           default: '{{repl LicenseFieldValue "geminiApiKey" }}'
 
+        ## Latitude
+        - name: latitudeApiKey
+          title: Latitude API key
+          type: password
+          when: '{{repl (ConfigOptionEquals "aiApiEnabled" "1") }}'
+          default: '{{repl LicenseFieldValue "latitudeApiKey" }}'
+
```

Comment:
If we have to provide that (that we're still discussing it AFAIK 😄) we'll have to provide it from our side, so let's don't even show it in the config